global-setup.ts

```diff
@@ -1,4 +1,6 @@
 import { mkdir, rm } from 'fs/promises';
+import startLocalRegistry from './tools/scripts/start-local-registry';
```

Comment:
This is unused here. Should this only be imported in global.e2e?

Reply:
I will remove it. Thanks